lib/jsonapi/utils/underscore.ex

```diff
@@ -0,0 +1,32 @@
+defmodule JSONAPI.Utils.Underscore do
+  @moduledoc """
+  Helpers for replacing underscores with dashes.
+  """
+
+  def underscore?, do: Application.get_env(:jsonapi, :underscore_to_dash, false)
+
+  def underscore(value) when is_atom(value) do
+    value
+    |> to_string
+    |> underscore
+  end
+
+  def underscore(value) when is_binary(value) do
+    value
+    |> String.replace("_", "-")
```

Comment:
We don't use pipes when there is a single input.

Reply:
fixed! 🙂